test/integration/data_entry_test.rb

```diff
@@ -16,7 +16,7 @@ class DataEntryTest < ActionDispatch::IntegrationTest
     before do
       # fill out the form
       select 'DC', from: 'patient_line'
-      fill_in 'Initial call date', with: 2.days.ago.strftime('%m/%d/%y')
+      fill_in 'Initial call date', with: 2.days.ago.strftime('%Y-%m-%d')
```

Comment:
Yeah -- if we have this as a text field instead of a date field, rails is looking for something in this format and it'll swallow it otherwise (or set it to 1900-01-01, I forget which). See #860 for the longer running issue on this.